src/main/java/com/example/travelservice/controllers/TravelerController.java

```diff
@@ -28,4 +28,18 @@ public Traveler createTraveler(@RequestBody TravelerDto travelerDto) {
         Traveler traveler = travelerConverter.convertTraveler(travelerDto);
         return travelerService.createTraveler(traveler);
     }
+
+    @PutMapping("traveler/id/{id}/update")
+    public Traveler updateEmail(@PathVariable Long id, @RequestBody String email){
+        return personalInfoService.updateEmail(id, email);
+    }
+
+    @PutMapping("traveler/id/{id}/update-model")
+    public Traveler updateEmail(@PathVariable Long id, @RequestBody TravelerDto travelerDto){
```

Comment:
If you are passing Traveler, then it's definitely not updateEmail

Reply:
method removed